kolibri/utils/options.py

```diff
@@ -49,6 +53,70 @@ def calculate_thread_pool():
     return MIN_POOL
 
 
+ALL_LANGUAGES = "all"
+SUPPORTED_LANGUAGES = "supported"
+BETA_LANGUAGES = "beta"
+
+
+def _set_en_first(value):
+    if "en" in value:
+        value.insert(0, value.pop(value.index("en")))
+
+
+def _coerce_language_value(value):
+    value = str(value)
+    if value == ALL_LANGUAGES:
+        value = list(KOLIBRI_LANGUAGE_INFO.keys())
+        if not value:
+            raise VdtValueError(ALL_LANGUAGES)
+        # For our default lists that contain en, promote it to the
+        # first so that it is the Django default language
+        _set_en_first(value)
+        return value
+    if value == SUPPORTED_LANGUAGES:
+        value = list(KOLIBRI_SUPPORTED_LANGUAGES)
+        if not value:
+            raise VdtValueError(SUPPORTED_LANGUAGES)
+        # For our default lists that contain en, promote it to the
+        # first so that it is the Django default language
+        _set_en_first(value)
+        return value
+    if value == BETA_LANGUAGES:
+        # en doesn't exist here
+        value = list(
+            set(KOLIBRI_LANGUAGE_INFO.keys()) - set(KOLIBRI_SUPPORTED_LANGUAGES)
+        )
+        if not value:
+            raise VdtValueError(BETA_LANGUAGES)
+        return value
+    return [value]
+
+
+def language_list(value):
+    """
+    Check that the supplied value is a list of languages,
+    or a single language, or a special shortcut parameter.
+    """
```

Comment:
Is this just a check, or does it do some kind of transformation?

It's not really clear to me what is "allowed" or what the API here is

Reply:
The library we are using does call it a check, but yes, it is also cleaning and changing the value to a standard output format.

Can add better comments.

Any ideas on how to best document the allowable values? (I think this would be helpful if we ever want to allow a UI to set these, as it would be useful to introspect them for the allowable values)